tmt/convert.py

```diff
@@ -95,16 +95,11 @@ def read_manual(plan_id, case_id, disabled):
         echo("Importing the '{0}' test case.".format(dir_name))
 
         # Test case data
-        md_content = {}
-
-        md_content['setup'] = html_to_markdown(testcase.setup)
-        md_content['action'] = html_to_markdown(testcase.action)
-        md_content['expected'] = html_to_markdown(testcase.effect)
-        md_content['cleanup'] = html_to_markdown(testcase.breakdown)
+        md_content = read_manual_data(testcase)
 
         # Test case metadata
         data = read_nitrate_case(testcase)
-        data['manual'] = True
+        data['manual'] = not testcase.automated
         data['test'] = 'test.md'
```

Comment:
It will always be `True`, right? So the original statement was probably a bit more clear.

Reply:
Probably, however if they aren't I wanted the variable to be consistent with the test selection (line 65).

Reply:
I mean, in the case selection you only choose tests which have `test.automated == False`. So here it always must be `True`. Or am I missing anything?

Reply:
Yeah, you are right.